rosidl_typesupport_connext_cpp/package.xml

```diff
@@ -11,15 +11,15 @@
   <buildtool_depend>connext_cmake_module</buildtool_depend>
   <buildtool_depend>rosidl_cmake</buildtool_depend>
   <buildtool_depend>rosidl_generator_c</buildtool_depend>
-  <buildtool_depend>rosidl_generator_cpp</buildtool_depend>
+  <buildtool_depend>rosidl_runtime_cpp</buildtool_depend>
```

Comment:
The package doesn't seem to b used in the `CMakeLists.txt`. Why is it needed at build time?

Reply:
https://github.com/ros2/rosidl_typesupport_connext/pull/49/commits/10197e939bac8025991db110642145638f8d9182